### ISSUE TITLE
-scanner with no scanners is a fatal error

### ISSUE THREAD

**Issue by briandfoy** [klep/scanline#39]
I just downloaded scanline and haven't figured out how to get it to recognize my scanner (not the reason for this report). Trying to list the scanners when none are found gives a fatal error. I would have expected perhaps that "No scanner was found." message again, or some help message saying that it's missing an argument:

```
$ scanline test
No scanner was found.
Done
$ scanline -scanner
Fatal error: Array index out of range: file /AppleInternal/BuildRoot/Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-1103.8.25.8/swift/stdlib/public/core/SwiftNativeNSArray.swift, line 77
```

**Comment by klep** [comment]
Hey @briandfoy -- Thanks for reporting this. It's expecting another parameter after "-scanner" (for example "-scanner Epson") but it shouldn't error out like that -- it should probably say that it's expecting another parameter.

I think maybe what you want is "scanline -list", which will show all the scanners that it's able to find on your network.

**Comment by briandfoy** [comment]
Yeah, I figured the rest of the stuff out. Just saw the weird case when I used it incorrectly. FWIW, I probably made that mistake because I'm used to see options that need a parameter listed differently, such as:

```
-scanner SCANNER
```

Not a big deal. I power cycled my scanner and it was recognized. That's nothing to do with scanline because Preview has the same problem sometimes.

Thanks for the nice tool :)
